sdk/python/feast/feature_store.py

```diff
@@ -1144,6 +1144,27 @@ def tqdm_builder(length):
                 feature_view, self.project, start_date, end_date,
             )
 
+    @log_exceptions_and_usage
+    def push(self, push_source_name: str, df: pd.DataFrame):
+        data_sources = self.registry.list_data_sources(self.project, True)
+        push_source = {
+            ds
+            for ds in data_sources
+            if isinstance(ds, PushSource) and ds.name == push_source_name
+        }
+
+        all_fvs = self.registry.list_feature_views(self.project, True)
+        fvs_with_push_sources = {
+            fv for fv in all_fvs if fv.stream_source in push_source
+        }
+
+        for fv in fvs_with_push_sources:
+            entities = []
+            for entity_name in fv.entities:
+                entities.append(self._registry.get_entity(entity_name, self.project))
```

Comment:
use `self.get_entity`

ideally also change the below code in `write_to_online_store` to use these FS methods instead of accessing registry directly

Reply:
done